setup.py

```diff
@@ -0,0 +1,22 @@
+from setuptools import setup
+
+install_requires = open('requirements.txt', 'r').readlines()
+
+setup(
+    name='apkleaks',
+    version='2.0.8',
```

Comment:
Is it possible to read from `VERSION` file instead?

Reply:
Please check the latest commit :)